README.md

```diff
@@ -27,12 +28,13 @@ A community made GraphQL API for Escape from Tarkov
 
 - View the prices of items
 - Get detailed ammo, armor, and weapon information
-- Fetch flea market data
+- Fetch real-time flea market data
```

Comment:
This isn't entirely accurate, probably the original description is more so.

Reply:
Reverted